src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -59,63 +53,77 @@ final class SettingsTable extends AbstractVirtualTable
                            .addPartitionKeyColumn(NAME, UTF8Type.instance)
                            .addRegularColumn(VALUE, UTF8Type.instance)
                            .build());
-        this.config = config;
+        Set<String> unprocessed = new HashSet<>(BACKWARDS_COMPATABLE_NAMES.values());
+        DatabaseDescriptor.visit((key, type, ro) -> {
+            if (BACKWARDS_COMPATABLE_NAMES.containsKey(key))
+            {
+                throw new AssertionError(String.format("Name '%s' is present in Config, this adds a conflict as this " +
+                                                       "name had a different meaning in the '%s",
+                                                       key, SettingsTable.class.getSimpleName()));
+            }
+            unprocessed.remove(key);
+        });
+        if (!unprocessed.isEmpty())
+            throw new AssertionError("The following keys must be present in the Config: " + unprocessed);
+    }
+
+    @Override
+    protected void applyColumnDeletion(AbstractMutableVirtualTable.ColumnValues partitionKey, AbstractMutableVirtualTable.ColumnValues clusteringColumns, String columnName)
+    {
+        String name = partitionKey.value(0);
+        String key = getKeyAndWarn(name);
+        runExceptionally(() -> DatabaseDescriptor.setProperty(key, null),
+                         t -> invalidRequest("Invalid delete request. Cause: %s", t.getMessage()));
+    }
+
+    @Override
+    protected void applyColumnUpdate(AbstractMutableVirtualTable.ColumnValues partitionKey,
+                                     AbstractMutableVirtualTable.ColumnValues clusteringColumns,
+                                     Optional<AbstractMutableVirtualTable.ColumnValue> columnValue)
+    {
+        String name = partitionKey.value(0);
+        String value = columnValue.map(v -> v.value().toString()).orElse(null);
+        runExceptionally(() -> DatabaseDescriptor.setProperty(getKeyAndWarn(name), value),
```

Comment:
this doesn't work, properties are not `String` in most cases, so you need to convert